webui/react/src/pages/ExperimentDetails/ExperimentVisualization/HpParallelCoordinates.tsx

```diff
@@ -56,6 +56,17 @@ interface HpTrialData {
   trialIds: number[];
 }
 
+export interface HermesInternalFilter {
+  p0: number;
+  p1: number;
+  value0: Primitive;
+  value1: Primitive;
+}
+
+export interface HermesInternalFilters {
+  [key: string]: HermesInternalFilter[];
+}
```

Comment:
on this section, I was hoping `Hermes.InternalFilter` and `Hermes.InternalFilters` would be exported like other types in this line: https://github.com/hkang1/hermes/pull/108/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R20

I wasn't able to import them though, and I couldn't guess if this is due to npm, typescript,  or these being `interface`s instead of `const`s.